client/src/pages/account/manage-resources/[id]/index.js

```diff
@@ -21,7 +21,7 @@ const ManageResource = ({ resourceId }) => {
       setResource(resourceRequest.isOk ? resourceRequest.response : false)
       const openRequest = await api.resources.requests.filter(
         resourceId,
-        { is_active: true },
+        { is_active_sharer: true },
         token
       )
       // if requests are appearing here that should be closed
```

Comment:
You can replace this comment with a new line now since it addresses the issue here.

I cant seem to select multiple lines of comments to suggest a change?